pkg/volume/plugins.go

```diff
@@ -183,9 +183,6 @@ type VolumeHost interface {
 	// directory might not actually exist on disk yet.
 	GetPodPluginDir(podUID types.UID, pluginName string) string
 
-	// GetKubeClient returns a client interface
```

Comment:
Whole point of this PR is to get rid of VolumeHost implementation from controllers. I don't want to create a dummy VolumeHost with fake methods like `GetPodPluginDir`.